### ISSUE TITLE
It starts to predict the same value as I increase sample number to predict

### ISSUE THREAD

**Issue by matteoottaviani** [GeoStat-Framework/PyKrige#123]
Hi, sorry for bothering you.
I have been dealing with the following problem for 3 months, hence I decided to try to share it.

I am training kriging and xgboost on sets of increasing sample number ( say from 100 samples to 1000 samples) of 15 inputs and 1 output each sample and I use the trained functions to predict a test set (that is always the same).

Whilst I haven't ever had any problem with xgboost prediction, when I predict the test set with kriging, I have no problems up to 400ish samples training; the more I increase from say around 400 samples to train, the more the last values of the test set which I predict equal the same value.
Have you got any idea about that?
Thanks!
Matteo

**Comment by MuellerSeb** [comment]
Could you provide a minimal example reproducing the problem?

**Comment by MuellerSeb** [comment]
Closing due to inactivity. PyKrige will be refactored either way. See #136.
Feel free to reopen.
